ships/0037-shipwright-build-strategy-controller.md

```diff
@@ -0,0 +1,190 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: shipwright-build-strategy-controller
+authors:
+  - "@jkhelil"
+reviewers:
+  - "@adambkaplan"
+  - "@qu1queee"
+approvers:
+  - "@adambkaplan"
+  - "@qu1queee"
+creation-date: 2023-10-02
+last-updated: 2023-10-02
+status: implementable
```

Comment:
@qu1queee for "implementable", should we require that the drawbacks and alternatives sections be filled in? These sections aren't marked as required in the current template.